pages/organizations/[organizationSlug]/activity.tsx

```diff
@@ -0,0 +1,328 @@
+  dataSource?: DataSource;
+  view?: View;
+  user: User;
+};
+
+const ActivityItem = ({
+  activity,
+  lastItem = false,
+}: {
+  activity: ActivityType;
+  lastItem?: boolean;
+}) => {
+  const avatarIcon = useMemo(() => {
+    switch (activity.action) {
+      case "create":
+        return <PlusIcon className="w-3 h-3 text-white" />;
+      case "bulkDelete":
+      case "delete":
+        return <TrashIcon className="w-3 h-3 text-white" />;
+      case "update":
+        return <RefreshIcon className="w-3 h-3 text-white" />;
+    }
+  }, [activity.action]);
+
+  const avatarBadgeColor = useMemo(() => {
+    switch (activity.action) {
+      case "create":
+        return "green";
+      case "bulkDelete":
+      case "delete":
+        return "red";
+      case "update":
+        return "blue";
+    }
+  }, [activity.action]);
+
+  const urlToSource = useMemo(
+    () =>
+      activity?.view
+        ? `/views/${activity.view.id}`
+        : `/data-sources/${activity?.dataSource?.id}/tables/${activity.tableName}`,
+    [activity?.view, activity?.dataSource, activity.tableName]
+  );
+
+  const urlToRecord = useMemo(
+    () =>
+      activity?.view
+        ? `${urlToSource}/records/${activity.recordId}`
+        : `${urlToSource}/${activity.recordId}`,
+    [urlToSource, activity.recordId]
+  );
+
+  const sourceName = useMemo(
+    () =>
+      activity?.view
+        ? activity?.view.name
+        : activity.dataSource?.name + "/" + activity.tableName,
+    [activity?.view, activity?.dataSource, activity.tableName]
+  );
+
+  const message = useMemo(() => {
+    switch (activity.action) {
+      case "create":
+        return (
+          <p>
+            Created record{" "}
+            <Code>
+              <a href={urlToRecord}>{activity.recordId}</a>
+            </Code>{" "}
+            in{" "}
+            <Code>
+              <a href={urlToSource}>{sourceName}</a>
+            </Code>
+          </p>
+        );
+      case "bulkDelete":
+        return (
+          <p>
+            Deleted{" "}
+            <Tooltip label={activity.recordId} placement="auto">
+              <u>multiple records</u>
+            </Tooltip>{" "}
+            from{" "}
+            <Code>
+              <a href={urlToSource}>{sourceName}</a>
+            </Code>
+          </p>
+        );
+      case "delete":
+        return (
+          <p>
+            Deleted record <Code>{activity.recordId}</Code> from{" "}
+            <Code>
+              <a href={urlToSource}>{sourceName}</a>
+            </Code>
+          </p>
+        );
+      case "update":
+        return (
+          <p>
+            Updated record{" "}
+            <Code>
+              <a href={urlToRecord}>{activity.recordId}</a>
+            </Code>{" "}
+            in{" "}
+            <Code>
+              <a href={urlToSource}>{sourceName}</a>
+            </Code>
+          </p>
+        );
+    }
+  }, [activity.action]);
+
+  const getChangeValue = (change: any) => {
```

Comment:
```suggestion
  const prettyValue = (change: any) => {
```
`prettyValue` says a bit more what it is...